UI/BindingSources/CSHARP/Binding/MainViewModel.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections.ObjectModel;
+using System.ComponentModel;
+
+namespace Binding;
+
+public class MainViewModel : INotifyPropertyChanged
+{
+	public event PropertyChangedEventHandler? PropertyChanged;
+
+	private ObservableCollection<Item>? _items;
+	public ObservableCollection<Item>? Items
+	{
+		get { return _items; }
+		set
+		{
+			_items = value;
+			OnPropertyChanged(nameof(Items));
+		}
+	}
+
+	public ICommand RemoveItemCommand { get; }
+
+	public MainViewModel()
+	{
+		Items = new ObservableCollection<Item>
+		{
+			new Item { Text = "Item 1" },
+			new Item { Text = "Item 2" },
+			new Item { Text = "Item 3" }
+		};
+
+		RemoveItemCommand = new RelayCommand(RemoveItem);
```

Comment:
If you want to use a Relay Command just use the CommunityToolkit

I'd rather see you use the generic version of the RelayCommand like:
```cs
RemoveItemCommand = new RelayCommand<Item>(RemoveItem, i => i is not null && Items.Contains(i));
```